packages/cxl-lumo-styles/scss/global.scss

```diff
@@ -79,6 +79,7 @@ ul {
  * headroom.js
  */
 .headroom {
+  top: var(--wp-admin--admin-bar--height, 0);
```

Comment:
Was this tested in logged out mode?

I don't see it being scoped in any way to admin-bar-only?

Reply:
> Was this tested in logged out mode?
> 
> I don't see it being scoped in any way to admin-bar-only?

After: https://github.com/WordPress/WordPress/commit/e32d9ec5226ec01e4a2de3cec4a4da0582d1d52c
Its automatically handled.

Logged out:
![logged-out-firefox_NTCdYkocxe](https://user-images.githubusercontent.com/1642796/177531725-954f42ec-31e8-47a7-95ea-4b58db29bea7.png)

Logged in:
![logged-in-firefox_nuz0HStY54](https://user-images.githubusercontent.com/1642796/177531757-26768b3f-842a-42e1-af44-28d0ef4b2a2e.png)

Reply:
Ah yeah, CSS var fallback should work.